File: Python Excel Examples/WorksheetsApiDemo/moveWorksheet.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "MoveWorksheet.xlsx"
folder = "ExcelDocument";
storage = ""
sheetName = "Sheet1"
moving =   spirecloudexcel.models.WorksheetMovingRequest()
moving.destination_worksheet = "Sheet4"
moving.position="after" # Can be Before, after
api.move_worksheet(name, sheetName, moving, folder=folder, storage=storage)

File: JavaScript Excel Examples/WorksheetsApiDemo/moveWorksheet.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorksheetsApi(apiClient);

    var name = "MoveWorksheet.xlsx";
    var opts = {
        folder: "input"
    };
    var sheetName = "Sheet1";
    var moving = new SpirecloudExcel.WorksheetMovingRequest();
    moving.destinationWorksheet = "sheet4";
    moving.position = "after";
    instance.moveWorksheet(name, sheetName, moving, opts, function (error, data, response) {
            if (error) throw error;
        });
})();